java/src/jmri/jmrit/beantable/signalmast/SignalMastTableDataModel.java

```diff
@@ -226,24 +245,11 @@ public void run() {
         javax.swing.SwingUtilities.invokeLater(t);
     }
 
-    @Override
-    public JTable makeJTable(@Nonnull String name, @Nonnull TableModel model, @Nullable RowSorter<? extends TableModel> sorter) {
-        JTable table = new SignalMastJTable(model);
-        table.setName(name);
-        table.setRowSorter(sorter);
-        table.getTableHeader().setReorderingAllowed(true);
-        table.setColumnModel(new XTableColumnModel());
-        table.createDefaultColumnsFromModel();
-
-        addMouseListenerToHeader(table);
-        return table;
-    }
-
     /**
      * Does not appear to be used.
      * 
      * @param srtr a table model
-     * @return a new table
+     * @return JTable a new table
```

Comment:
The type of return value is automatically completed by the JavaDocs. We don't want to repeat it here.